recipes/kssd/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0" %}
+
+package:
+  name: kssd
+  version: {{ version }}
+
+source:
+  url: https://github.com/yhg926/public_kssd/archive/v{{ version }}.tar.gz
+  sha256: a5dcaf520049a962bef625cb59a567ea2b4252d4dc9be28dd06123d340e03919
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - zlib
```

Comment:
Is this statically linking zlib in?

Reply:
It looks like this makes system calls to gzip (it incorrectly reads in the zlib headers and then doesn't use them). This is fine then.